lib/schedule/gtfs/route.ex

```diff
@@ -48,10 +54,23 @@ defmodule Schedule.Gtfs.Route do
         0 => route_directions[0] && route_directions[0].direction_name,
         1 => route_directions[1] && route_directions[1].direction_name
       },
+      type: type,
       name: name
     }
   end
 
+  @spec row_has_route_type?(Csv.row()) :: boolean()
+  @doc """
+  Verify that "route_type" exists on the row, especially to prevent issues while testing
+  """
+  def row_has_route_type?(%{"route_type" => nil}) do
+    raise ArgumentError, message: "route_type is required on route rows"
+  end
+
+  def row_has_route_type?(_route_row) do
+    true
+  end
```

Comment:
**question:** what about the case where `"route_type"` isn't a key in the map? I suppose that won't happen as long as there is a `route_type` column in the CSV but it might still be worth guarding against. I think rewriting it like so would take care of it:
```suggestion
  def row_has_route_type?(%{"route_type" => route_id}) when not is_nil(route_id) do
    true
  end

  def row_has_route_type?(_route_row) do
    raise ArgumentError, message: "route_type is required on route rows"
  end
```